browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -1049,6 +1049,20 @@ index 78980810141a9e9b65b93e6cebe80daff9a52dc1..28fdad87ee06982ed8826df77e6ba933
              ]
 +        },
 +        {
++          "name": "willRequestOpenWindow",
++          "description": "Fired when page tries to open a new window.",
++          "parameters": [
++              { "name": "url", "type": "string" }
```

Comment:
I wonder if we need some identifier of the document opening the window to attribute that to the page before/after navigation